src/network/networkd-ipv6-proxy-ndp.c

```diff
@@ -38,7 +38,7 @@ static int ipv6_proxy_ndp_address_configure(Link *link, const struct in6_addr *a
         if (r < 0)
                 return log_link_error_errno(link, r, "Could not create RTM_NEWNEIGH message: %m");
 
-        r = sd_rtnl_message_neigh_set_flags(req, NLM_F_REQUEST | NTF_PROXY);
+        r = sd_rtnl_message_neigh_set_flags(req, NTF_PROXY);
```

Comment:
`NLM_F_REQUEST`  is not associated with ndp. It's must be inn all request messages in netlink
https://git.kernel.org/pub/scm/network/iproute2/iproute2.git/tree/ip/ipneigh.c#n113

Reply:
`NLM_F_REQUEST` is set by default in `message_new()` which is called in `sd_rtnl_message_new_neigh()`. So, it is not necessary to set explicitly here.

Reply:
Umm then we should fix the commit message according to that

Reply:
Hmm? The problem here is that `NLM_F_REQUEST` is a nlmsg flag and used in `sd_netlink_message_set_flags()`. Should not used in `sd_rtnl_message_neigh_set_flags()`. I already commented so.

Reply:
Yeah that's right